docs/Upgrading to CSLA 9.md

```diff
@@ -174,3 +174,66 @@ CSLA 9 supports the use of nullable reference types in your code. This means tha
 Supporting nullable types means that some APIs have changed to support nullable types.
 
 * The `User` and `Principal` properties of `ApplicationContext` no longer return null
+
+#### Using Timeout in `HttpProxy` and `HttpCompressionProxy`
+
+##### Overview
+
+The `HttpProxy` and `HttpCompressionProxy` classes now support configuring timeouts through the `HttpProxyOptions` class. This allows for more flexible and centralized configuration of HTTP request timeouts.
+
+##### Configuration
+
+To configure the timeout for `HttpProxy` and `HttpCompressionProxy`, you need to set the `Timeout` and `ReadWriteTimeout` properties in the `HttpProxyOptions` class.
+
+##### Example
+
+Below is an example of how to configure and use the timeout settings in `HttpProxy` and `HttpCompressionProxy`.
+
+###### Step 1: Configure `HttpProxyOptions`
```

Comment:
Seeing this. We should extend the `HttpProxyOptionsExtensions` to set these values with the fluent api.

Reply:
Something like this:
```csharp
services.AddCsla(
    o => o.DataPortal(
        o2 => o2.AddClientSideDataPortal(
            cso => cso.UseHttpProxy(
                hpo => hpo.WithTimeout(TimeSpan.FromSeconds(30)).WithReadWriteTimeout(TimeSpan.FromSeconds(30))
            )
        )
    )
);
```

Reply:
I added this code, check it first and then I change the docs

Reply:
The code looks good. Please adjust the docs 👍

Reply:
Changed, i hope